DataFormats/METReco/interface/HcalPhase1FlagLabels.h

```diff
@@ -0,0 +1,32 @@
+#ifndef DataFormats_METReco_HcalPhase1FlagLabels_h
+#define DataFormats_METReco_HcalPhase1FlagLabels_h
+
+#include "DataFormats/METReco/interface/HcalCaloFlagLabels.h"
+
+namespace HcalPhase1FlagLabels
+{
+    enum HBHEStatusFlag {
+        /*  0 */ HBHEHpdHitMultiplicity = HcalCaloFlagLabels::HBHEHpdHitMultiplicity,
+        /* 11 */ HBHEIsolatedNoise = HcalCaloFlagLabels::HBHEIsolatedNoise,
+        /* 12 */ HBHEFlatNoise = HcalCaloFlagLabels::HBHEFlatNoise,
+        /* 13 */ HBHESpikeNoise = HcalCaloFlagLabels::HBHESpikeNoise,
+        /* 15 */ HBHETS4TS5Noise = HcalCaloFlagLabels::HBHETS4TS5Noise,
```

Comment:
We will see what happens... There is no decision yet on how 2016
HCAL will be handled. For now, both options are open -- we can
switch or we can maintain the legacy code.

On 08/04/2016 05:47 PM, Slava Krutelyov wrote:
>
> In DataFormats/METReco/interface/HcalPhase1FlagLabels.h <https://github.com/cms-sw/cmssw/pull/15324#discussion_r73616084>:
>
> > @@ -0,0 +1,32 @@
> > +#ifndef DataFormats_METReco_HcalPhase1FlagLabels_h
> > +#define DataFormats_METReco_HcalPhase1FlagLabels_h
> > +
> > +#include "DataFormats/METReco/interface/HcalCaloFlagLabels.h"
> > +
> > +namespace HcalPhase1FlagLabels
> > +{
> > +    enum HBHEStatusFlag {
> > +        /*  0 */ HBHEHpdHitMultiplicity = HcalCaloFlagLabels::HBHEHpdHitMultiplicity,
> > +        /* 11 */ HBHEIsolatedNoise = HcalCaloFlagLabels::HBHEIsolatedNoise,
> > +        /* 12 */ HBHEFlatNoise = HcalCaloFlagLabels::HBHEFlatNoise,
> > +        /* 13 */ HBHESpikeNoise = HcalCaloFlagLabels::HBHESpikeNoise,
> > +        /* 15 */ HBHETS4TS5Noise = HcalCaloFlagLabels::HBHETS4TS5Noise,
>
> if you plan to change meaning/naming of the same enum values, what is your plan to support 2016 HCAL ?
> If both 2016 and future phase1 will be handled by HcalPhase1FlagLabels, it's probably OK
> (HcalCaloFlagLabels will just become deprecated and eventually go away).
>
> Please clarify.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub <https://github.com/cms-sw/cmssw/pull/15324/files/f2734c7b29b226480f1efc887de88c8ae73f121f#r73616084>, or mute the thread 
> <https://github.com/notifications/unsubscribe-auth/AFS4-Sz8C7nvIJGSchAD0Ih6qEMHhtG5ks5qcmvwgaJpZM4JXio9>.
>